packages/frint-cli/commands/new.js

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable no-multi-spaces, no-use-before-define */
+
+const mkdirp = require('mkdirp');
+const request = require('request');
+const tar = require('tar');
+
+const createApp = require('frint').createApp;
+
+const DEFAULT_ORGANIZATION = 'Travix-International';
+const DEFAULT_REPOSITORY = 'frint';
+const DEFAULT_BRANCH = 'master';
+const DEFAULT_EXAMPLES_DIR = 'examples';
+const DEFAULT_EXAMPLE = 'counter';
+const DESCRIPTION_TEXT = `
+Usage:
+
+  $ frint new
+  $ frint new <directory>
+  $ frint new <directory> --example <example>
+
+Example:
+
+  $ frint new myapp --example kitchensink
+  $ frint new myapp --example frint-vue/tree/master/examples/basic
+
+You can find a list of all available official examples here:
+https://github.com/Travix-International/frint/tree/master/examples
+`.trim();
+const INVALID_EXAMPLE_ARG_TEXT = `
+Invalid <example> value. Must be in one of the following formats:
+
+  * <name>
+  * <organization>/<repository>/tree/<branch>/**
+`.trim();
+
+module.exports = createApp({
+  name: 'new',
+  providers: [
+    {
+      name: 'summary',
+      useValue: 'Scaffolds a new Frint app in specified directory',
+    },
+    {
+      name: 'description',
+      useValue: DESCRIPTION_TEXT,
+    },
+    {
+      name: 'execute',
+      useFactory: function useFactory(deps) {
+        return function execute() {
+          deps.console.log('Initializing...');
+          Promise.resolve(deps)
+            .then(mapDepsToContext)
+            .then(createOutputDirectory)
+            .then(streamExampleToOutputDirectory)
+            .then(() => deps.console.log('Done!'))
```

Comment:
we can have some additional info here. like asking the user to run two more commands to start their application:

```
Done!

Please run these two commands to start your application:

  $ npm install
  $ npm start
```